backend/src/main/java/codezap/template/domain/Template.java

```diff
@@ -79,9 +75,11 @@ public void updateTemplate(String title, String description, Category category,
         this.visibility = visibility;
     }
 
-    public void validateAuthorization(Member member) {
-        if (!getMember().equals(member)) {
-            throw new CodeZapException(ErrorCode.FORBIDDEN_ACCESS, "해당 템플릿에 대한 권한이 없습니다.");
-        }
+    public boolean matchMember(Member member) {
+        return getMember().equals(member);
+    }
```

Comment:
일부러 getMember 사용하신건가요?
```suggestion
    public boolean matchMember(Member member) {
        return this.member.equals(member);
    }
```